### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by chumaachike**
Made Changes

### CODE REVIEW

**Review by chumaachike — COMMENTED** [review]
Nice Job so far!
Great job setting up linters perfectly ✔
 I there needs to be a couple of changes to make the code better
Please ensure you have no linter errors in your code
I think the code needs more  styling like background color, color, to make the code look like the template in the freecodecamp website.

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
          <section class="main-section" id="Java"><header >Java</header>
```
Comment (chumaachike) [inline index.html:18]: I think you should consider removing the header tag from this section as it is not semantically correct.

**Review by chumaachike — COMMENTED** [review]
##  Nice Job!!
However there a few suggested changes I have 
### Good work setting up the linters however ensure that there are no linter errors on the code

### Nice job creating a style page. Please endeavor to style the page like the template on freecodecamp

**Review by aimemalaika — APPROVED** [review]
Hi @chumaachike, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**